scIB/integration.py

```diff
@@ -21,6 +21,7 @@
 import rpy2.robjects as ro
 import anndata2ri
 from scipy.sparse import issparse
+import SAUCIE
```

Comment:
let's only import the methods within the function so that we don't require them all to be installed when scIB is installed.